impeller/entity/contents/contents.cc

```diff
@@ -86,10 +90,17 @@ std::optional<Snapshot> Contents::RenderToSnapshot(
       [&contents = *this, &entity, &coverage](const ContentContext& renderer,
                                               RenderPass& pass) -> bool {
         Entity sub_entity;
+        auto color_size = contents.GetColorSourceSize();
+        if (color_size.has_value()) {
+          auto sx = coverage->size.width / color_size->width;
```

Comment:
Nit: I believe sizes have `operator/` already.

Reply:
So the origin correction is the stuff missing here right?

Reply:
Using the `operator/` now.

Adding the origin correction was making things wrong on Metal. Do we still need it for Vulkan?

Reply:
Vulkan and Metal coordinate spaces should be configured to be the same (after the vk_khr_maintenance1).